feature_engine/selection/drop_psi_features.py

```diff
@@ -577,10 +582,24 @@ def _get_cut_off_value(self, split_column):
 
         return cut_off
 
-    def _calculate_auto_threshold(self, N, M, q=0.999):
-        # threshold = χ2(q,B−1) × (1/N + 1/M)
-        # where q - quantile (or 1 - p-value) B - number of bins,
-        # N - size of basis dataset, M - size of test dataset
-        # see formula (5.2) from reference
-        # taking q = 0.999 to get higher threshold
-        return stats.chi2.ppf(q, self.bins - 1) * (1.0 / N + 1.0 / M)
+    def _calculate_auto_threshold(self, N, M, bins):
+        """Threshold computation for chi-square test.
```

Comment:
could you also write docstring for the arguments?

Reply:
Why are we changing the default parameters of the auto-threshold? We used to default q to 0.999 and now this is an obligatory parameter.

What's the logic?

fyi @glevv

Reply:
Wrote some form of docstring for the arguments, but not sure if it's in the format desired. Please let me know if you want it to be something else. 

The logic behind changing the arguments is that the auto-threshold is called on by the class, but the p-value (or quantile, as previously used) could not be defined by the user. I would deem this to be a necessary option in order to make use of the auto-threshold (which I think is very useful in and of itself). 

The bins argument needs to be added for auto-threshold to be compatible with categorical features. For details, please see my discussion with Claudio in the Issues thread.